README.md

```
@@ -34,7 +34,7 @@ bun run init # bun install && bun husky:prepare
 
 1. To run:
 
-```bash
+```bash    
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)